content/overview/kubedb/databases/kafka.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Run and Manage Kafka Database on Kubernetes - KubeDB
```

Comment:
Change the title to -
`Run and Manage Apache Kafka on Kubernetes - KubeDB`
As kafka is not actually a database.

Reply:
Resolved